### ISSUE TITLE
`/cglow entities` takes ~100ms to execute, causing FPS to tank with an active `/cglow --keep-searching-entities`

### ISSUE THREAD

**Issue by James103** [Earthcomputer/clientcommands#390]
As of Client Commands 2.6.5 for Minecraft 1.18.2, running `/cglow entities` almost always takes approximately 100ms to execute.

In addition, having an active `/cglow --keep-searching-entities` task running causes the render thread's FPS to drop to a maximum of 1-2, with each frame taking ~900ms to execute.

According to profiler results, when I ran `/cglow entities @e[type=zombie]` many times during the ten seconds after pressing F3+L, they took a significant amount of render time, at 24% of the 10-second interval (i.e. 2.4 seconds of rendering time spent executing these commands). After running `/cglow --keep-searching-entities entities @e[type=zombie]`, pressing F3+L, then waiting for 10 seconds, inspecting the profiler results gives 94.86% of the 11120-ms interval spent executing that task alone.

For comparison, on the server side, running `/effect give @e[type=minecraft:zombie] minecraft:glowing` every tick only added about 1 mspt to the server tick time, whereas the client-side version added ~800-900 mspt.

Mod list:
```
  [✔️] clientcommands-2.6.5
  [✔️] fabric-api-0.47.8+1.18.2
  [✔️] lazydfu-0.1.2
```

The complete MultiMC profile is attached:
[1.18.2 Testing.zip](https://github.com/Earthcomputer/clientcommands/files/8452204/1.18.2.Testing.zip)

**Comment by Earthcomputer** [comment]
Probably related to the switch to clientarguments.

**Comment by Earthcomputer** [comment]
Fixed in 2.6.6
